index.html

```diff
@@ -37,7 +37,160 @@ <h1>I'm Newton <br>Glad to see you!</h1>
+            </li>
+            <li class="categories">
+                <img src="./img/portfolio-tonic2.png" alt="Tonic Category Image">
+                <div class="left-block">
+                    <h2 class="project-title">
+                        Tonic
+                    </h2>
+                    <span class="canopy">
+                        CANOPY
+                        <ul>
+                            <li><img src="./img/counter.png" alt=""></li>
+                            <li>Back End Dev</li>
+                            <li><img src="./img/counter.png" alt=""></li>
+                            <li>2015</li>
+                        </ul>
+                    </span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="portfolio-tags">
+                        <li>
+                            html
+                        </li>
+                        <li>
+                            css
+                        </li>
+                        <li>
+                            javascript
+                        </li>
+                    </ul>
+                    <a href="" class="project-links">See Project</a>
+                </div>
+            </li>
+            <li class="categories">
+                <img src="./img/portfolio-stories2.png" alt="Tonic Category Image">
+                <div class="left-block">
+                    <h2 class="project-title">
+                        Multi-Post Stories
+                    </h2>
+                    <span class="canopy">
+                        CANOPY
+                        <ul>
+                            <li><img src="./img/counter.png" alt=""></li>
+                            <li>Back End Dev</li>
+                            <li><img src="./img/counter.png" alt=""></li>
+                            <li>2015</li>
+                        </ul>
+                    </span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="portfolio-tags">
+                        <li>
+                            html
+                        </li>
+                        <li>
+                            css
+                        </li>
+                        <li>
+                            javascript
+                        </li>
+                    </ul>
+                    <a href="" class="project-links">See Project</a>
+                </div>
+            </li>
+        </ul>
+    </section>
+    <section class="about-myself">
+        <h1>About Myself</h1>
+        <p >Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <span>
+            <p class="lets-connect">LET'S CONNECT</p>
+        
```

Comment:
```
<p class="lets-connect">LET'S CONNECT</p>
```

- [ ] Please let us use a header tag `hx` here instead of `p` tag for semantic purposes.